packages/react-components/react-tree/src/components/TreeItem/TreeItem.types.ts

```diff
@@ -1,5 +1,6 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
-import { TreeItemContextValue } from '../../contexts/treeItemContext';
+import type { TreeItemContextValue } from '../../contexts';
+import type { ButtonContextValue } from '@fluentui/react-button';
```

Comment:
_nit:_ Keep relative imports after absolute ones

 ```suggestion
import type { ButtonContextValue } from '@fluentui/react-button';
import type { TreeItemContextValue } from '../../contexts';
```